src/solver/optim-model-filler/include/antares/solver/optim-model-filler/TimeDependentLinearExpression.h

```diff
@@ -56,10 +57,10 @@ class TimeDependentLinearExpression
     TimeDependentLinearExpression operator/(const TimeDependentLinearExpression& other) const;
     /// Multiply linear expression by -1
     TimeDependentLinearExpression operator-() const;
-    const std::map<unsigned, LinearExpression>& GetLinearExpressions() const;
+    const std::unordered_map<unsigned, LinearExpression>& GetLinearExpressions() const;
     size_t getSize() const;
 
 private:
-    std::map<unsigned int, LinearExpression> linearExpressions_;
+    std::unordered_map<unsigned int, LinearExpression> linearExpressions_;
```

Comment:
careful when changing a `std::map` to `std::unordered_map`, the order is no longer guaranteed e.g
```cpp
for (auto [k, v] : m)
{

}
```
does not loop in ascending key order for `std::unordered_map`. This should be fine, but it's a risk of error of we rely on keys being ordered.